src/readers/src/virtex_data_provider.h

```diff
@@ -6,21 +6,23 @@
 
 namespace Readers {
 
+
+// FIXME we should return one instance FileRepresentation only
```

Comment:
Please do it

Reply:
Not now. It is not priority one task. Moreover idk this interface